packages/core/upload/package.json

```diff
@@ -50,6 +51,7 @@
     "react-redux": "8.0.5",
     "react-select": "5.7.0",
     "sharp": "0.32.0",
```

Comment:
Do you think we could bump sharp to 0.33.0? I'm thinking of this PR https://github.com/strapi/strapi/pull/19311 and the attached issue.

Reply:
yes! lets do that

Reply:
Any updates on this?